samples/bluetooth/central_hr_coded/README.rst

```diff
@@ -13,7 +13,7 @@ However, this sample specifically looks for heart rate monitors using LE Coded P
 Overview
 ********
 
-The sample demonstrates a Bluetooth LE Central role functionality by scanning for other Bluetooth LE devices that run a Heart Rate Server with LE Coded PHY support.
+The sample demonstrates a Bluetooth LE Central role functionality by scanning for other Bluetooth LE devices that run a Heart Rate Server with LE Coded PHY support which not available in ZephyrLL
```

Comment:
I think the official term for the Zephyr LL is this: `Zephyr Bluetooth LE Controller` - as it is written in _doc/nrf/ug_ble_controller.rst_ - you can link to that label.